bin/build-modules.js

```diff
@@ -31,7 +31,13 @@ function buildPackage(pkg) {
 }
 
 readDir('packages/node_modules').then(function (packages) {
-  return Promise.all(packages.map(buildPackage)).catch(function (err) {
+  var promise = Promise.resolve();
+  packages.forEach(function (pkg) {
+    promise = promise.then(function () {
+      return buildPackage(pkg);
+    });
+  });
+  return promise.catch(function (err) {
```

Comment:
For the life of me I can't tell why, but this parallel build process stopped working with my commit. Maybe there's a dependency issue. Maybe I was trying to be too clever. Who knows.

The build is a bit slower now, but it works! shrugmoji